hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/AlertsHandler.java

```diff
@@ -115,11 +115,15 @@ public Response findAlerts(
                 allowableValues = "Comma separated list of [LOW, MEDIUM, HIGH, CRITICAL]")
             @QueryParam("severities")
             final String severities,
-            @ApiParam(required = false, value = "Filter out events for unspecified tags.",
+            @ApiParam(required = false, value = "Filter out alerts for unspecified tags.",
                 allowableValues = "Comma separated list of tags, each tag of format 'name\\|value'. + \n" +
                         "Specify '*' for value to match all values.")
             @QueryParam("tags")
             final String tags,
+            @ApiParam(required = false, value = "Filter out alerts for unspecified tags.",
+                allowableValues = "A tag query expression.")
```

Comment:
+1
Yes, I put a temporal description here but not filled.
I will try to summarize the BNF grammar (at least an example) and see how it fits in the asciidoc render.